### ISSUE TITLE
Websocket persistent notification: show a dynamic string from the server instead of static message

### ISSUE THREAD

**Issue by mattdm** [home-assistant/android#2268]
**Is your feature request related to a problem? Please describe.**

The new websocket persistent notification actually provides a nice, quick shortcut to Home Assistant. However, after seeing it for a few hours, being told "Listening to Home Assistant via Websocket" is no longer really useful information. Further, while it's convenient to have "Websocket settings" there initially, that is not something one will normally want to change frequently.

I know the persistent notification must stay in order to keep Android from killing the app under pressure. That's fine. But as it is, the notification is using some of the most valuable and scarce UI real estate (and visible/attention space) to just show a basically useless static message and a rarely-useful configuration link.

This is part of several suggestions to improve this (linked by #2266).

**Describe the solution you'd like**

The text "Listening to Home Assistant via Websocket" should instead be set to a string fetched from the HA server. This could be implemented in a number of different ways:

1. A new property of the Mobile App integration, set by a template
2. A configuration value for the Mobile App integration, pointing to some _other_ template entity which would return a string
3. Something else. Really, the exact server-side implementation is not particularly important to me, and I'm new to the overall architecture so there may be a great idea I have no idea about.

Regardless of how this is implemented, a template seems important — it should be easy for the HA admin to customize the message based on their own setup and desires.

**Describe alternatives you've considered, if any**

See parent issue and other related suggestions.

**Additional context**

As noted, following on from https://github.com/home-assistant/android/issues/2266#issuecomment-1037329212_

**Comment by dshokouhi** [comment]
So if you want a server side message stored in HA core then you need to first create a HA core feature request on the forums. Then once implemented we can accept a feature request in the app. The developers working on the android app are not the same as HA core developers.  I am going to close this request and we can revisit once the HA core side is ready.

**Comment by mattdm** [comment]
Hmmm; are you disconnected from the Mobile App integration in HA core? That's a little surprising. Having it there would require a core change. But I guess there's another way — you could just let the mobile app user select an entity and get the string from that, right? There doesn't really need to be anything specially about it on the server side. (Am I missing something? Very possible!)

**Comment by dshokouhi** [comment]
Then in this case this becomes a duplicate of #2266 as that wants to use entity states or a template. Both of which would be configured from the app.

When you say server side that implies the mobile app integration to pick things up as that is how the app communicates to HA.

We are not disconnected from mobile app but this app is dependent on changes done to that integration.

**Comment by mattdm** [comment]
> Then in this case this becomes a duplicate of #2266 as that wants to use entity states or a template. Both of which would be configured from the app.

That issue is quite general (and I don't think it mentions entity states, even?). This is meant to be a concrete suggestion for one specific thing which would go to the general goal in that higher-level request. I hope that makes sense and is helpful. If you'd rather treat it as a duplication of #2266, that's cool with me too. :)

**Comment by dshokouhi** [comment]
Your request and the other request mention using templates for the notification, to me that feels like a duplicate. The app will just render the template and display its text based on the request.
